pkgs/applications/misc/sweethome3d/default.nix

```diff
@@ -100,7 +99,7 @@ let
       inherit description;
       inherit license;
       maintainers = [ lib.maintainers.edwtjo ];
-      platforms = lib.platforms.linux;
+      platforms = jdk.meta.platforms;
```

Comment:
nit
```suggestion
      inherit (jdk.meta) platforms;
```

Reply:
Rewrote it to have explicit list of supported platform. I tested it on `aarch64-darwin`, thus added that as well.